extensions/functions_comparison.yaml

```diff
@@ -83,4 +83,36 @@ scalar_functions:
       - args:
           - value: fp64
         return: BOOLEAN
+  -
+    name: "is_finite"
+    description: Whether a value is finite.
```

Comment:
This should probably describe the meaning of finite here. For example, is NaN finite?

Reply:
Updated now.